packages/graphql-codegen-core/src/codegen.ts

```diff
@@ -298,7 +298,7 @@ function validateDuplicateDocuments(files: Types.DocumentFile[]) {
             `.trimRight();
             })
             .join('')}
-    `.trimRight()
+    `.trimEnd()
```

Comment:
`trimRight` is deprecated

Reply:
@B2o5T we should add ESlint rule for this